internal/controller/replication.storage/parameters.go

```diff
@@ -53,10 +55,12 @@ func validatePrefixedParameters(param map[string]string) error {
 		if strings.HasPrefix(k, replicationParameterPrefix) {
 			switch k {
 			case prefixedReplicationSecretNameKey:
+			case prefixedGroupReplicationSecretNameKey:
```

Comment:
I think you need to do

```go
			switch k {
			case prefixedReplicationSecretNameKey:
				fallthrough
			case prefixedGroupReplicationSecretNameKey:
				if v == "" {
...
```

Reply:
Interestingly, when I tested my changes it was working fine without the `fallthrough` keyword 🤔

Reply:
Okay, I just check on a code locally, and seems like it is needed. Not sure, why didn't it compain during the testing.
Added it now.